### PR TITLE
E2E: Makes the test get to the redirect every time

### PR DESCRIPTION
**Pull request by aidvu**
was flaky, as the scrolling thing doesn't seem to work great with new Playwright...

### CODE REVIEW

**Comment by aidvu** [comment]
Now only to figure why `route.abort()` breaks the test...

File: test/e2e/specs/onboarding/signup__with-theme-LOHP.ts [hunk 1]
```diff
@@ -78,15 +78,16 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 					themeButtonUrl.protocol = calypsoUrl.protocol;
 
 					await route.abort();
+					await page.waitForTimeout( 2000 );
```
Comment (aidvu) [inline test/e2e/specs/onboarding/signup__with-theme-LOHP.ts]: And this lets `abort()` finish, so no race condition with `page.goto()`... hell.

File: test/e2e/specs/onboarding/signup__with-theme-LOHP.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 
 			// Hovering over the theme card is necessary to make the "Start with this theme" button visible.
 			const themeCard = themeContainer.locator( '.lp-image-top-row' ).last();
-			await themeCard.hover();
+			await themeCard.hover( { force: true } );
```
Comment (tyxla) [inline test/e2e/specs/onboarding/signup__with-theme-LOHP.ts]: Added some [forcing](https://playwright.dev/docs/input#forcing-the-click) to some of the actions to make them more reliable.

File: test/e2e/specs/onboarding/signup__with-theme-LOHP.ts [hunk 3]
```diff
@@ -74,19 +74,17 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 			if ( calypsoUrl.hostname !== 'wordpress.com' ) {
 				// Reroute the click to the current Calypso URL.
 				await page.route( themeButtonUrl.href, async ( route ) => {
-					themeButtonUrl.host = calypsoUrl.host;
-					themeButtonUrl.protocol = calypsoUrl.protocol;
-
 					await route.abort();
-					await page.goto( themeButtonUrl.href );
```
Comment (tyxla) [inline test/e2e/specs/onboarding/signup__with-theme-LOHP.ts:81]: A `goto` right after `abort` will obviously break things. Moving it out of the `route()` completely.

File: test/e2e/specs/onboarding/signup__with-theme-LOHP.ts [hunk 4]
```diff
@@ -74,19 +74,17 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
...
+				themeButtonUrl.protocol = calypsoUrl.protocol;
+				await page.goto( themeButtonUrl.href );
 			}
 			// Get theme slug
 			const pageMatch = new URL( themeButtonUrl.href ).search.match( 'theme=([a-z]*)?&' );
 			themeSlug = pageMatch?.[ 1 ] || null;
-
-			// Hover, otherwise the element isn't considered stable, and is out of the viewport.
-			await themeCard.hover();
+			await themeCard.hover( { force: true } );
```
Comment (tyxla) [inline test/e2e/specs/onboarding/signup__with-theme-LOHP.ts:90]: More force hover.

**Review by tyxla — APPROVED** [review]
Go for it 👍

**Pull request merged**